ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2181,6 +2209,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-upi
+- as: baremetalds-assisted-ovn-hypershift-agent-mgmt-p2-f7
```

Comment:
Suggest to remove `ovn` accordingly in the job name.

Reply:
Thanks for your remind, Done